src/Avalonia.OpenGL/GlVersion.cs

```diff
@@ -11,12 +11,15 @@ public record struct GlVersion
         public GlProfileType Type { get; }
         public int Major { get; }
         public int Minor { get; }
+        public bool EnableCompatibilityProfile { get; } // Only makes sense if Type is OpenGL and Version is >= 3.2
```

Comment:
Note that `GlVersion` struct is used not only for specifying the desired capabilities, but to report the properties of an existing context. So the property should be named `IsCompatibilityProfile`

Reply:
I renamed the property. And, right, it is used in `AvaloniaNativeGlPlatformGraphics`, so I altered code here as well so that `IsCompatibilityProfile` is correctly reported.